README.md

```diff
@@ -38,27 +42,38 @@ for hands-on MLOps tutorials using CML!
 
 ## Table of contents
 
-1. [Usage](#usage)
-2. [Getting started (tutorial)](#getting-started)
-3. [Using CML with DVC](#using-cml-with-dvc)
-4. [Using self-hosted runners](#using-self-hosted-runners)
-5. [Install CML as a package](#install-cml-as-a-package)
-6. [Example Projects](#see-also)
+1. [Setup (GitLab, GitHub, Bitbucket)](#setup)
+2. [Usage](#usage)
+3. [Getting started (tutorial)](#getting-started)
+4. [Using CML with DVC](#using-cml-with-dvc)
+5. [Advanced Setup (Self-hosted, local package)](#advanced-setup)
+6. [Example projects](#see-also)
 
-## Usage
+## Setup
 
-You'll need a GitHub or GitLab account to begin. Users may wish to familiarize
-themselves with [Github Actions](https://help.github.com/en/actions) or
+You'll need a GitHub, GitLab, or Bitbucket account to begin. Users may wish to
+familiarize themselves with [Github Actions](https://help.github.com/en/actions)
+or
 [GitLab CI/CD](https://about.gitlab.com/stages-devops-lifecycle/continuous-integration).
 Here, will discuss the GitHub use case.
 
-- **GitLab users**: Please see our
-  [docs about configuring CML with GitLab](https://github.com/iterative/cml/wiki/CML-with-GitLab).
-- **Bitbucket Cloud users**: Please see our
-  [docs on CML with Bitbucket Cloud](https://github.com/iterative/cml/wiki/CML-with-Bitbucket-Cloud).
-  _Bitbucket Server support estimated to arrive by May 2021._
-- **GitHub Actions users**: The key file in any CML project is
-  `.github/workflows/cml.yaml`:
+### GitLab
```

Comment:
> Awesome! Is there any specific reason to have them sorted like GitLab>Bitbucket>GitHub? It doesn't seem to be alphabetical nor have anything to do with popularity.

I had a vague idea with GL>BB>GH.

1. GL: currently slightly more used so should be first
2. GH: these docs are on GH so people will assume it's supported so we don't have to advertise it so can put it last

In general cases however (e.g. cml.dev) point (2) doesn't apply so I would go with GH>GL>BB.

Reply:
Follow-up of https://github.com/iterative/cml/pull/617#discussion_r660638668